dataline-tests/src/acceptanceTests/java/io/dataline/test/acceptance/AcceptanceTests.java

```diff
@@ -75,61 +82,169 @@
 @SuppressWarnings("rawtypes")
 public class AcceptanceTests {
 
-  static final Logger LOGGER = LoggerFactory.getLogger(AcceptanceTests.class);
+  private static final Logger LOGGER = LoggerFactory.getLogger(AcceptanceTests.class);
+  private PostgreSQLContainer sourcePsql;
+  private PostgreSQLContainer targetPsql;
 
-  static PostgreSQLContainer SOURCE_PSQL;
-  static PostgreSQLContainer TARGET_PSQL;
-
-  DatalineApiClient apiClient = new DatalineApiClient(
+  private DatalineApiClient apiClient = new DatalineApiClient(
       new ApiClient().setScheme("http")
           .setHost("localhost")
           .setPort(8001)
           .setBasePath("/api"));
 
-  @BeforeAll
-  public static void init() throws IOException, InterruptedException {
-    SOURCE_PSQL = new PostgreSQLContainer();
-    TARGET_PSQL = new PostgreSQLContainer();
-    SOURCE_PSQL.start();
-    TARGET_PSQL.start();
+  private List<UUID> sourceImplIds = Lists.newArrayList();
```

Comment:
isn't it safer to always assign it to a new array in the `init`?

Reply:
Yup, done